README.md

```diff
@@ -120,6 +121,7 @@ jobs:
                   IOS_PACKAGE: 'ios.mock'
                   ANDROID_PACKAGE: 'android.mock'
                   NOWSECURE_GROUP_ID: ${{ secrets.NOWSECURE_GROUP_ID }}
+                  SEVERITY_LIST: ''
```

Comment:
Maybe an example like 

```
SEVERITY_LIST: 'medium, want'
```

might be more useful to the end user.

Reply:
update to SEVERITY_LIST: 'high, medium'